index.js

```diff
@@ -30,10 +31,11 @@ function autoContentType (req, file, cb) {
   file.stream.once('data', function (firstChunk) {
     var type = fileType(firstChunk)
     var mime
+    var svgMatches = firstChunk.toString().match(svgRegex) || []
```

Comment:
Use `.test` to get a boolean instead of an array ☺️ 

```suggestion
    var svgMatches = svgRegex.test(firstChunk.toString())
```

Reply:
does `.toString()` always work on a `Buffer`, even if it contains invalid unicode codes? 🤔

Reply:
It passes on the PNG test, when it converts the PNG buffer to string it just contains invalid characters.

Reply:
![image](https://user-images.githubusercontent.com/8794965/54306659-1b27b780-4587-11e9-9a0a-471df948f855.png)